paymentapi/src/main/java/fi/hel/verkkokauppa/payment/service/refund/PaytrailRefundPaymentService.java

```diff
@@ -245,47 +244,44 @@ private RefundPayment createRefundPayment(
         return refundPayment;
     }
 
-    public void triggerRefundPaymentPaidEvent(RefundPayment payment) {
+    public void triggerRefundPaymentPaidEvent(RefundPayment refundPayment) {
         String now = DateTimeUtil.getDateTime();
 
-        PaymentMessage.PaymentMessageBuilder paymentMessageBuilder = PaymentMessage.builder()
-                .eventType(EventType.PAYMENT_PAID)
-                .eventTimestamp(now)
-                .namespace(payment.getNamespace())
-                .paymentId(payment.getRefundPaymentId())
-                .orderId(payment.getOrderId())
-                .userId(payment.getUserId())
-                .paymentPaidTimestamp(now)
-                .orderType(payment.getRefundType());
+        RefundMessage.RefundMessageBuilder refundMessageBuilder = RefundMessage.builder()
+                .eventType(EventType.REFUND_PAID)
+                .timestamp(now)
+                .namespace(refundPayment.getNamespace())
+                .refundId((refundPayment.getRefundId()))
+                .refundId(refundPayment.getRefundPaymentId())
```

Comment:
This should probably be .paymentId(refundPayment.getRefundPaymentId()) ?

Reply:
Double refundId - whoops. You are right, I'll fix this.